internal/os_updater/ubuntu/update.go

```diff
@@ -118,7 +122,7 @@ func sizeToBytes(size string, unit string) uint64 {
 	}
 }
 
-const noUpdateAvailable = "0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded."
+const noUpdateAvailable = "0 upgraded, 0 newly installed, 0 to remove"
```

Comment:
Any reason why we remove "and 0 not upgraded."?

Reply:
@yengliong93 - I had a scenario on my system where that equaled 1 and it wouldn't give the no upgrade needed remark and pass with 200.  The only way I could get it to work was to remove the check for not upgraded.  Since that has to do upgrades instead of updates, I was thinking that might be alright.  What are your thoughts?

Reply:
I see. The "1 not upgraded" is not being upgraded could be due to several reasons:

- The package might be held back due to a package pinning configuration.
- There might be dependency issues preventing the upgrade.
- The package might be part of a larger upgrade that requires additional steps or manual intervention.